src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,120 @@
+.container {
+  @extend %font-style;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  padding: 50px 40px;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  margin-right: 48px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n+4) {
+    margin-top: 48px;
+  }
+
+  &__photo {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    margin-top: 4px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__action {
+    position: relative;
+    margin-top: 17px;
+  }
+
+  &__review {
+    position: absolute;
+    top: 50%;
+    transform: translate(0, -50%);
+    right: 0;
+    font-size: 10px;
+    line-height: 14px;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__buy {
+    padding: 11px 0;
+    margin-top: 16px;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    background: #00acdc;
+    border-radius: 5px;
+    border: 1px solid transparent ;
+  }
+
+  &__buy:hover {
+    color: #00acdc;
+    background-color: #fff;
+    border: 1px solid #00acdc;
+  }
+}
+
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(n+5) {
```

Comment:
stars should have modifiers as in the previous tasks
<img width="584" alt="image" src="https://user-images.githubusercontent.com/102411130/222367438-fdaaeee1-2582-4631-9670-117e660fd2f4.png">